src/CMakeLists.txt

```diff
@@ -912,8 +913,9 @@ if(WITH_LIBCEPHFS OR WITH_KRBD)
   add_library(parse_secret_objs OBJECT ${parse_secret_files})
 endif()
 
+add_subdirectory(client)
```

Comment:
@tchaikov 
Yes, I was first hidding under `WITH_LIBCEPFS` but I needed it for WITH_FUSE.
There is however nothing very OS or LIB dependant stuff in there, so I figured to build it always.
Could conditionalise, if you prefer.